src/python/visclaw/ipyclaw.py

```diff
@@ -6,15 +6,23 @@
 """
 
 from __future__ import absolute_import
-import sys
-import clawpack.visclaw.Iplotclaw as Iplotclaw
+from argparse import ArgumentParser
+from clawpack.visclaw.Iplotclaw import Iplotclaw
 
-def run_iplotclaw(setplot='setplot.py',outdir='./_output'):
-    ip = Iplotclaw.Iplotclaw(setplot=setplot,outdir=outdir)
+def run_iplotclaw(**kwargs):
+    ip = Iplotclaw(**kwargs)
     ip.plotloop()
 
+def parse_args():  # TODO: ipython cannot parse custom arguments
```

Comment:
We should be able to call this using Python so I am not sure I understand why IPython is an issue.

Reply:
Ok I'm just a noob, I've just found out about the `-i --` flag. Removing it now.